modules/virt-troubleshooting-vmware-vm-import.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+[id='virt-troubleshooting-vmware-vm-import_{context}']
+= Troubleshooting a VMware virtual machine import
+
+If an imported virtual machine's status is `Import error: (VMware)`, you can check the Conversion Pod log for errors:
+
+. Obtain the Conversion Pod name:
++
+----
+$ oc get pods -n <project> | grep v2v <1>
+kubevirt-v2v-conversion-f66f7d-zqkz7            1/1     Running     0          4h49m
+----
+<1> Specify the project of your imported virtual machine.
+
+. Obtain the Conversion Pod log:
++
+----
+$ oc logs kubevirt-v2v-conversion-f66f7d-zqkz7 -f -n <project>
+----
+
+[id='error-messages_{context}']
+== Error messages
+
+* If an imported virtual machine event displays the error message, `Readiness probe failed`, the following error message appears in the Conversion Pod log:
++
+----
+INFO - have error: ('virt-v2v error: internal error: invalid argument: libvirt domain ‘v2v_migration_vm_1’ is running or paused. It must be shut down in order to perform virt-v2v conversion',)"
+----
++
+You must ensure that the virtual machine is shut down before importing it.
+
+[id='known-issues_{context}']
+== Known issues
+
+* Your environment must have sufficient storage space for the imported disk.
++
+If you try to import a virtual machine whose disk size is larger than the available storage space, the operation cannot complete. You will not be able to import another virtual machine or to clean up the storage because there are insufficient resources to support object deletion. To resolve this situation, you must add more object storage devices to the storage backend. link:https://bugzilla.redhat.com/show_bug.cgi?id=1721504[(*BZ#1721504*)]
```

Comment:
s/whose/with a disk size that is

Reply:
I fixed that in a different PR. :-) It hasn't been merged yet. I guess I could make the same change here.

Reply:
Odd. I did fix that and the change was merged. Although I rebased this branch, the update doesn't appear.